spec/controllers/mailings_controller_spec.rb

```diff
@@ -6,17 +6,14 @@
   let(:mailing)        { FactoryGirl.create(:mailing) }
   let(:user)           { FactoryGirl.create(:user) }
 
-  shared_context 'User logged in' do
-    before { sign_in user }
-  end
-
   describe 'GET index' do
     before { mailing }
```

Comment:
I'm afraid I don't understand this pattern. What's the difference to making line 6 a `let!(...)`?